src/cli/commands/colorcheck.rs

```diff
@@ -1,11 +1,12 @@
 use crate::commands::prelude::*;
 use crate::hdcanvas::Canvas;
+use crate::output::Output;
```

Comment:
We can move the `use crate::output::Output;` import to `commands::prelude`. This way, we don't have to include it for every single command.